.circleci/config.yml

```diff
@@ -207,17 +207,14 @@ jobs:
                --label "build.number=$CIRCLE_BUILD_NUM" \
                --label "build.url=$CIRCLE_BUILD_URL" \
                --label "build.gitref=$CIRCLE_SHA1"
-      - when:
-          condition: << parameters.main >>
-          steps:
-            - run:
-                name: Publish to docker hub
-                command: |
-                  docker login -u="${QUAYIO_USERNAME}" -p="${QUAYIO_PASSWORD}" quay.io
-                  docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
-                  docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:latest
-                  docker push quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
-                  docker push quay.io/hmpps/<<parameters.app_name>>:latest
+      - run:
+          name: Publish to docker hub
+          command: |
+            docker login -u="${QUAYIO_USERNAME}" -p="${QUAYIO_PASSWORD}" quay.io
+            docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
+            docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:latest
+            docker push quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
+            docker push quay.io/hmpps/<<parameters.app_name>>:latest
```

Comment:
This will push branch builds with tag `latest`
Comment this line as well when you want a branch docker image and revert once done.